corehq/util/workbook_reading/adapters/csv.py

```diff
@@ -36,8 +36,10 @@ def __init__(self, csv_file):
     def to_workbook(self):
         rows = []
 
-        # Loop through the rows, and add each row's contents to rows.
+        # Loop through the rows, and add each row's contents to rows, making sure
+        # not escape '\u' characters.
         for row in csv.reader(self._file, delimiter=","):
-            rows.append(row)
+            row_decoded = [column.encode().decode('unicode-escape') for column in row]
```

Comment:
@millerdev could you confirm this looks right to you?

Reply:
I think `.encode()` will use the default encoding, which may not be dependable. I can't remember if it's always UTF-8 on Python 3, but probably better to be explicit.
```suggestion
            row_decoded = [column.encode("utf-8").decode('unicode-escape') for column in row]
```
I'm not sure of the reason for `.decode('unicode-escape')`. @dchukhin can you elaborate on your reasoning here?

It might make more sense to open the file (`self._file`) with `encoding="utf-8"` and remove this line.